src/main/java/com/trivialTrivia/persistence/Trivias.java

```diff
@@ -0,0 +1,36 @@
+package com.trivialTrivia.persistence;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.trivialTrivia.api.TriviaApi;
+import com.trivialTrivia.entity.Trivia;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.JSONArray;
+import org.json.JSONObject;
+@Path("/trivias")
+public class Trivias  {
+    private Logger logger = LogManager.getLogger(this.getClass());
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getTrivias(@QueryParam("triviaCategory") String category, @QueryParam("triviaAmount") String amount,
```

Comment:
@cruzang000 Just confirming- amount is supposed to be a string? I saw this and got confused, but then I checked your api class and it's also a string, so this must be correct.

Reply:
ETA- I'm using int amountOfQuestions in the entity